database/migrations/studentstable.js

```diff
@@ -0,0 +1,26 @@
+
+exports.up = function(knex, Promise) {
+	
+	return knex.schema.createTable('students', function(tbl) {
+	
+		tbl.increments();
+	
+		tbl.string('name', 100);
+	
+		tbl
+
+		.integer('cohort_id')
+
+		.unsigned()
+
+		.references('id')
+
+		.inTable('cohorts');
+	});
+};
+
+exports.down = function(knex, Promise) {
+
+	return knex.schema.dropTableIfExists('students');
+
+};
```

Comment:
https://github.com/Spencer-M24/webdb-iii-challenge/blob/145bccba7a0df484f01526a3f7804789e4472855/database/migrations/studentstable.js#L2-L26

Looks great - nice use of the references between tables.